include/seqan3/range/views/minimiser.hpp

```diff
@@ -503,13 +503,13 @@ namespace seqan3::views
  * \{
  */
 
-/*!\brief                   Computes minimisers for a range of type std::size_t. A minimiser is the smallest value in a
+/*!\brief                   Computes minimisers for a range of type std::integral. A minimiser is the smallest value in a
```

Comment:
... for a range of integral numbers. (std::integral is not a type but a concept of a group of types with certain properties)